scripts/gpt/preprocess.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -ev
+
+
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ -z $2 ]]; then
```

Comment:
Can you explain these args somewhere? Follow-up PR is fine but the way I'm reading it is:

1 arg provided: $1: GCS location to upload processed training data.
2 args: $1: GCS location where training data is stored. $2: GCS location to upload processed training data.

Reply:
Added usage insturctions